### ISSUE TITLE
Deterministically Hash Samples to Determine Validation Set

### ISSUE THREAD

**Issue by rameshvarun** [rameshvarun/cs231n-project#1]
Right now Keras automatically sets the last 10% of data to be the validation set - https://github.com/fchollet/keras/issues/597

This should be randomized, but also deterministic (so that the same examples are always in the validation set).
